Source/mpqapi.h

```diff
@@ -28,7 +28,7 @@ struct _FILEHEADER {
 struct _HASHENTRY {
 	Uint32 hashcheck[2];
 	Uint32 lcid;
-	Uint32 block;
+	int32_t block;

```

Comment:
This changed signedness, was this intended?

Reply:
Yes, it is checked for -1 and -2